studio/components/interfaces/Account/AuditLogs.tsx

```diff
@@ -0,0 +1,293 @@
+import * as Tooltip from '@radix-ui/react-tooltip'
+import dayjs from 'dayjs'
+import { useEffect, useState } from 'react'
+
+import { ScaffoldContainerLegacy } from 'components/layouts/Scaffold'
+import Table from 'components/to-be-cleaned/Table'
+import AlertError from 'components/ui/AlertError'
+import { DatePicker } from 'components/ui/DatePicker'
+import ShimmeringLoader from 'components/ui/ShimmeringLoader'
+import { AuditLog } from 'data/organizations/organization-audit-logs-query'
+import { useOrganizationsQuery } from 'data/organizations/organizations-query'
+import { useProfileAuditLogsQuery } from 'data/profile/profile-audit-logs-query'
+import { useProjectsQuery } from 'data/projects/projects-query'
+import { Alert, Button, IconArrowDown, IconArrowUp, IconRefreshCw } from 'ui'
+import FilterPopover from '../Organization/AuditLogs/FilterPopover'
```

Comment:
Reusing these files from `Organization` folder. Is there a better place for them?

Reply:
yeap - shifted them to interfaces/AuditLogs